src/ui/ui_manager.py

```diff
@@ -602,6 +642,75 @@ def repair_needed(self) -> bool:
             use_grayscale=True)
         return template_match.valid
 
+    def gambling_needed(self) -> bool:
+        return self._gold_full
+    
+    def set__gold_full (self, bool: bool):
+        self._gold_full = bool
+        self._gambling_round = 1
+
+    def gamble(self, item_finder: ItemFinder):
+        gold = True
+        gamble_on = True
+        if self._config.char["num_loot_columns"]%2==0:
+            ignore_columns = self._config.char["num_loot_columns"]-1
+        else:
+            ignore_columns = self._config.char["num_loot_columns"]-2
+        template_match = self._template_finder.search_and_wait(
+                "REFRESH",
+                best_match=True,
+                threshold=0.79,
+                time_out=4)
+        if template_match.valid:
+            #Gambling window is open. Starting to spent some coins
+            while (gamble_on and gold):
+                if (self._inventory_has_items (self._screen.grab(),self._config.char["num_loot_columns"], ignore_columns) and self._inventory_has_items (self._screen.grab(),2)):   
+                    gamble_on = False
+                    self.close_vendor_screen ()
+                    break
+                for item in self._config.gamble["gamble_items"]:  
+                    template_match_item = self._template_finder.search (item.upper(), self._screen.grab(), roi=self._config.ui_roi["vendor_stash"])
+                    while not template_match_item.valid:
+                        #Refresh gambling screen
+                        template_match = self._template_finder.search ("REFRESH", self._screen.grab())
+                        if (template_match.valid):
+                            x, y = self._screen.convert_screen_to_monitor(template_match.position)
+                            mouse.move(x, y, randomize=12, delay_factor=[1.0, 1.5])
+                            wait(0.1, 0.15)
+                            mouse.click(button="left")
+                            wait(0.1, 0.15)
+                        template_match_item = self._template_finder.search (item.upper(), self._screen.grab(), roi=self._config.ui_roi["vendor_stash"])
+                    #item found in gambling menu
+                    x, y = self._screen.convert_screen_to_monitor(template_match_item.position)
```

Comment:
anywhere you're using `convert_screen_to_monitor()` on a template match, you can simply add `normalize_monitor = True` to the arguments of `template_finder.search()` and then do `mouse.move(*template_match.position , ...` See #499